app/assets/stylesheets/icons.scss

```diff
@@ -1,23 +1,12 @@
 @import 'variables';
 
 .brand-icon { height: 25px; }
-.icon-small { height: 20px; }
```

Comment:
These styles were all for the old `Icon` class. They're not being used in the new component. The new `.icon-button` class is what resets the default button styling for use with icons in a `button_to`.